cccatalog-api/test/api_live_integration_test.py

```diff
@@ -75,6 +75,23 @@ def test_image_detail(search_fixture):
     response = requests.get(API_URL + '/image/{}'.format(test_id), verify=False)
     assert response.status_code == 200
 
+def test_image_delete(search_fixture):
+    test_id = search_fixture['results'][0]['id']
+    should_fail = requests.delete(
+        f'{API_URL}/image/{test_id}',
+        auth=('invalid', 'credentials'),
+        verify=False
+    )
+    assert should_fail.status_code == 401
+    response = requests.delete(
+        f'{API_URL}/image/{test_id}',
+        auth=('continuous_integration', 'deploydeploy'),
+        verify=False
+    )
+    assert response.status_code == 204
+    deleted_response = requests.get(f'{API_URL}/image/{test_id}')
+    assert deleted_response.status_code == 404
```

Comment:
Just nitpicking here, but is it worth breaking this up into 3 different tests?

Reply:
I wondered the same while writing it. I do think smaller unit tests are better so I went ahead and split them up.